example/platform.html

```diff
@@ -97,8 +97,9 @@
 <script>
   function initViewer() {
     var viewerHost = document.getElementById('viewerHost');
-    var ampUrl = 'https://www.ampproject.org/';
-    var viewer = new AmpViewer(viewerHost, ampUrl);
+    var ampUrl = 'https://www-ampproject-org.cdn.ampproject.org/v/s/www.ampproject.org/?amp_js_v=0.1#origin=http://localhost:8000';
```

Comment:
Most of the URL needs to be constructed by the `AmpViewer` instance itself right before attachment. That would include `amp_js_v`, `#origin` and all the rest. Very likely the cache URL as well (or the fact whether or not the cache URL is used).